validator/validator-main.protoascii

```diff
@@ -3839,6 +3839,9 @@ tags: {
 # 4.8 Links
 # Links are a concept, rather than a tag. See tagspecs for "LINK", "A", etc.
 
+# 4.8.11 The canvas element
+# See extensions/amp-mustache/validator-amp-script.protoascii
```

Comment:
I think Greg means moving the entire canvas tag spec here with a comment for mandatory_ancestor.

Reply:
right. He is also fine with this approach as long as one as point out in main where the rule is defined. Thanks